### DISCUSSION TITLE
Please design custom variables for the default layer as OpenStreetMap,

### DISCUSSION THREAD

**Discussion by dfwsbylwy** [opengisch/QField#3382] [Ideas]
Before, when a new data package was opened, Qfield would add the default layer openstreetmap(### _QStringLiteral( "type=xyz&url=https://a.tile.openstreetmap.org/%7Bz%7D/%7Bx%7D/%7By%7D.png&zmax=19&zmin=0&crs=EPSG3857" ), QStringLiteral( "OpenStreetMap" ), QLatin1String( "wms" ) )_) to the layer. However, I hope that the XYZ online layer network address of this default layer can be customized, which will be more suitable, convenient and wider for users.

**Comment by nirvn** [comment]
@dfwsbylwy , we've got something ever better for you, fully customized project as base map: https://docs.qfield.org/how-to/standalone-datasets/#using-a-project-as-base-map-for-standalone-datasets
